059-statically-configured-atc/proposal.md

```diff
@@ -0,0 +1,192 @@
+# Summary
+
+Static Configuration of Concourse Instances
+
+# Motivation
+
+Running Concourse at scale, for a long period of time, poses a number of
+problems that make operability difficult.
+
+* In large organizations, it is not feasible for a team to set up Concourse by
+  themselves. If Concourse adoption is being driven by the engineering team
+  itself (i.e. bottom-up adoption), then the team is often challenged to
+  orchestrate the acquisition of servers (for `web` and for workers), storage, a
+  PostgreSQL database, networking, and credential access. If Concourse adoption
+  is being driven by engineering leadership (i.e. top-bottom adoption), then
+  leadership is challenged to orchestrate the synchronization of configuration
+  across a wide variety of Concourse instances that have been set up for
+  engineering teams.
+* For any sufficiently long-lived Concourse instance, database usage grows over
+  time, and the underlying PostgreSQL database grows vertically to match,
+  increasing the costs for end teams. Reducing reliance upon the underlying
+  PostgreSQL database has a direct, positive effect on instance performance and
+  costs incurred by Concourse administrators.
+* Perusing the current configuration for a Concourse instance, when so much of
```

Comment:
I'm not exactly sure what this point means, is this talking about the `web` configuration of the postgres instance (https://github.com/concourse/concourse/blob/master/atc/atccmd/command.go#L130) or the configuration of postgres itself (which is done depending how you deployed it through helm, bosh, etc).